s5_base/local/samromur_data_prep.sh

```diff
@@ -15,28 +15,33 @@ set -e
 
 data_type=$2
 data_dir=$3
-audio_src_dir=$1/audio/
-metadata=$1/metadata_v2.tsv
-
+audio_src_dir=$1/audio
```

Comment:
Why did you remove the trailing forward slash?

Reply:
because when you concatenate it later in the code it, you use / again making // appear. Linux will process this but still an 'error
'